tests/Integration/Bridge/TolerantParser/SourceCodeFilesystem/ScfClassCompletorTest.php

```diff
@@ -38,16 +39,85 @@ public function provideComplete(): Generator
                     'type' => Suggestion::TYPE_CLASS,
                     'name' => 'Alphabet',
                     'short_description' => 'Test\Name\Alphabet',
+                    'class_import' => 'Test\Name\Alphabet',
+                ],
+                [
+                    'type' => Suggestion::TYPE_CLASS,
+                    'name' => 'Backwards',
+                    'short_description' => 'Test\Name\Backwards',
+                    'class_import' => 'Test\Name\Backwards',
+                ],
+                [
+                    'type' => Suggestion::TYPE_CLASS,
+                    'name' => 'Clapping',
+                    'short_description' => 'Test\Name\Clapping',
+                    'class_import' => 'Test\Name\Clapping',
+                ],
+                [
+                    'type' => Suggestion::TYPE_CLASS,
+                    'name' => 'WithoutNS',
+                    'short_description' => 'WithoutNS',
+                    'class_import' => null,
+                ],
+            ],
```

Comment:
If possible I would rather these tests were as minimal as possible, i.e. we are testing lots of things here, but we only want to assert that suggestions are provided for `extends`

Reply:
Do the additional suggestions add any value to _this_ test?